src/gui/tray/usermodel.cpp

```diff
@@ -658,7 +658,7 @@ QString User::statusEmoji() const
 
 bool User::serverHasUserStatus() const
 {
-    return _account->account()->capabilities().userStatusNotification();
+    return _account->account()->capabilities().userStatus();
```

Comment:
should we not use:
```
return _account->account()->capabilities().userStatus() && account->account()->capabilities().userStatusNotification();
```

Reply:
Why? The docs just say we should check the capabilities. We don't check the endpoints for other capabilities.

Reply:
OK